public/video-ui/src/services/pandaReqwest.js

```diff
@@ -1,15 +1,15 @@
-import reqwest from "reqwest";
-import { reEstablishSession } from "babel-loader?presets[]=es2015!panda-session";
-import { getStore } from "../util/storeAccessor";
+import reqwest from 'reqwest';
+import { reEstablishSession } from 'babel-loader!panda-session';
```

Comment:
`@babel/preset-env` handles this stuff now, so no need to set the preset in the import